in-adapter-api/src/main/java/gohigher/auth/support/TokenType.java

```diff
@@ -0,0 +1,7 @@
+package gohigher.auth.support;
+
+public enum TokenType {
+
+	ACCESS,
+	REFRESH
```

Comment:
```suggestion
	ACCESS,
	REFRESH,
	;
```
별건 아니지만.. 토큰 타입이 추가될일은 없지만... 어떠신가요...

Reply:
좋아요!!